packages/core/src/tools/mcp-client.ts

```diff
@@ -125,6 +139,15 @@ export function getMCPServerStatus(serverName: string): MCPServerStatus {
   );
 }
 
+/**
+ * Get the discovered prompts for an MCP server
+ */
+export function getMCPServerPrompts(
```

Comment:
It's already on the mcp-client

```suggestion
export function getPrompts(
```